contrib/julia-config.jl

```diff
@@ -26,28 +26,8 @@ end
 private_libDir() = joinpath(JULIA_HOME, Base.PRIVATE_LIBDIR)
 
 function includeDir()
-    joinpath(match(r"(.*)(bin)",JULIA_HOME).captures[1],"include","julia")
-end
-
-function unixInitDir()
-    filePart = split(imagePath(),"/")[end]
-    return match(Regex("(.*)(/julia/$filePart)"),imagePath()).captures[1]
-end
-
-function windowsInitDir()
-    if imagePath()[end-1:end] == "ji"
-        return match(r"(.*)(\\julia\\sys.ji)",imagePath()).captures[1]
-    else
-        return match(r"(.*)(\\julia\\sys.dll)",imagePath()).captures[1]
-    end
-end
-
-function initDir()
-    if is_unix()
-        return unixInitDir()
-    else
-        return windowsInitDir()
-    end
+    dir, bin = splitdir(JULIA_HOME)
+    return joinpath(dir, "include", "julia")
```

Comment:
this PR isn't making it worse, but this is incorrect if the layout has been set differently, like in a distro build